### ISSUE TITLE
Issue with 👷‍♂️

### ISSUE THREAD

**Issue by mehdicopter** [arvinxx/gitmoji-commit-workflow#576]
Hello,

I found a bug using the emoji 👷‍♂️

When I use the code the test passes

```
echo ':construction_worker: ci: test' | commitlint # passes
```

But when I use the emoji itself it fails

```
 echo '👷 ci: test' | commitlint
⧗   input: 👷 ci: test
✖   👷 is not in the correct gitmoji list, please check the emoji code on https://gitmoji.dev/. [start-with-gitmoji]

✖   found 1 problems, 0 warnings
ⓘ   Get help: https://github.com/conventional-changelog/commitlint/#what-is-commitlint
```

It happens with several other emojis as well

**Comment by mehdicopter** [comment]
I believe this is related to the [regex](https://github.com/arvinxx/gitmoji-commit-workflow/blob/b30226810fdad6d314b902b5c4b459bd0bfe7b9c/packages/commitlint-plugin/src/rule.ts#L10)

**Comment by mehdicopter** [comment]
It does work when I did fix the code within the [json file](https://raw.githubusercontent.com/carloscuesta/gitmoji/master/src/data/gitmojis.json)

```json
{
  "emoji": "👷",
  "entity": "&#x1f477;",
  "code": ":construction_worker_man:",
  "description": "Add or update CI build system.",
   "name": "construction-worker",
   "semver": null
},
```

I did change `:construction_worker:` into `:construction_worker_man:`

You can verify it by this code snippet

```javascript
const name = require("emoji-name-map");
const unicode = require("emoji-unicode-map");

console.log(name.get(":construction_worker:")); // undefined
console.log(unicode.get("👷")); // construction_worker_man
```

**Comment by arvinxx** [comment]
fixed in `commitlint-plugin-gitmoji@2.2.5`
